### DISCUSSION TITLE
How to duplicate tour?

### DISCUSSION THREAD

**Discussion by gregoryxxxxxx** [mytourbook/mytourbook#935] [Q&A]
Hey! Y'all moved! 

I know there was talk of how to do this in the past - and I am still doing a duplicate by using the "unique ID" thing. But... is there any option for duplicating an existing tour that isn't manual at this point? Or still the same as past? 

Thx!

**Comment by wolfgang-ch** [comment]
I created a new feature request https://github.com/mytourbook/mytourbook/issues/937
